src/Init/Tactics.lean

```diff
@@ -941,6 +1008,61 @@ If there are several with the same priority, it is uses the "most recent one". E
 ```
 -/
 syntax (name := simp) "simp" (Tactic.simpPre <|> Tactic.simpPost)? (ppSpace prio)? : attr
+
+
+/-- The possible `norm_cast` kinds: `elim`, `move`, or `squash`. -/
```

Comment:
It'd be great if this docstring had a forward reference to the `norm_cast` docstring that describes them.